network.c

```diff
@@ -1082,6 +1082,30 @@ struct iio_context * network_create_context(const char *host)
 		ret = getaddrinfo(addr_str, port_str, &hints, &res);
 	} else {
 		ret = getaddrinfo(host, IIOD_PORT_STR, &hints, &res);
```

Comment:
Just a quick question. Would it be better to resolve first with `dnssd_resolve_host`, instead of calling it on fallback?

Reply:
Hmm, never thought about that as I just wanted to be the less disruptive possible. I would assume that most of the more popular linux distros already come with `nss-mdns` which means `getaddrinfo` is capable of resolving the name [and so we should really get into the fallback phase] and most likely faster than this implementation. OTOH, using `dnssd_resolve_host` first should ensure that things always work without the need of a fallback... I have no preference here...

Reply:
Could you try if `dnssd_resolve_host` work with an IP and with `localhost`? If it does, then we can call it first, then fall back to getaddrinfo.